tests/components/islamic_prayer_times/test_init.py

```diff
@@ -100,10 +97,7 @@ async def test_islamic_prayer_times_timestamp_format(hass: HomeAssistant) -> Non
         await hass.config_entries.async_setup(entry.entry_id)
         await hass.async_block_till_done()
 
-        assert (
-            hass.data[islamic_prayer_times.DOMAIN].prayer_times_info
-            == PRAYER_TIMES_TIMESTAMPS
-        )
+        assert hass.data[islamic_prayer_times.DOMAIN].data == PRAYER_TIMES_TIMESTAMPS
```

Comment:
Tests will be updated as part of PR #85518 as the library has been updated and it now returns iso formatted datetime value. This will be done once this PR is merged. I will remove these assertion and assert the sensor states instead.

Reply:
Ok.